drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/widget/FactPatternWidget.java

```diff
@@ -936,4 +937,14 @@ private String getRuleDialect() {
         }
         return DRLConstraintValueBuilder.DEFAULT_DIALECT;
     }
+
+    public Connectives getConnectives() {
+        return connectives;
+    }
+
+    public CEPOperatorsDropdown getNewOperatorDropdown(String[] operators,
```

Comment:
Did you want this to ideally be package protected?

Reply:
Same as above